mqtt.go

```diff
@@ -147,6 +147,8 @@ func (client *MQTTClient) Run() {
 		}
 	}
 
+	client.Publish("parasite-scanner/status", "online", false, 1)
```

Comment:
I think we can set the retained flag to true here (changing the `false` to `true`).

This retained flag tells the MQTT broker to remember this value, so it will also send it to new clients. If we set to `false`, as in here, if we restart Home Assistant, it won't see anything in the `parasite-scanner/status` topic.

By using the retained flag `true`, MQTT will always remember this value, and when Home Assistant is restarted and re-checks this topic, the value "online" will still be there like we want.

Let me know if that makes sense!